### DISCUSSION TITLE
Add German DFB Pokal

### DISCUSSION THREAD

**Discussion by Kayinwonderland** [vasqued2/ha-teamtracker#105] [Ideas]
Hey there,

is it possible to also add CUps, like the German DFB Pokal ( https://www.espn.co.uk/football/league/_/name/ger.dfb_pokal) to the Team Tracker? I am just working with conditional cards  and  plan on showing the one that is upccoming next 

Thanks a lot

**Comment by vasqued2** [comment]
Use a custom API configuration
```
- platform: teamtracker
  league_id: XXX
  team_id: RBL
  sport_path: soccer
  league_path: ger.dfb_pokal
  name: pokal_rb_leipzig
```

**Reply by Kayinwonderland** [reply]
Thanks a lot 👍

**Comment by Nniikka** [comment]
That's what I need @
